add_new_host.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+load 'ambari_client.rb'
+require 'yaml'
+require 'byebug'
+
+# Parse command line.
+cluster_name = ARGV[0]
+datanode = ARGV[1]
+
+# Load node attributes.
+node_attr = YAML.load_file('cluster.yml')[0]
+
+# Connect to ambari server.
+connection = node_attributes['ambari_server']
+
+# Create a new node with something like:
```

Comment:
go ahead and remove these comment lines, they are roadsigns and placeholders for when you are writing the code. The final code should only include comments that help a person understand how the code works or what it's doing,